### PR TITLE
Create 2023Q3.adoc

### PR DESCRIPTION
**Pull request by mehmetoguzderin**
Let's iterate together, this is just a very initial draft (maybe not even that, just creating the file)! We can bring up during the meeting if time allows.

### CODE REVIEW

**Comment by mehmetoguzderin** [comment]
Tentatively disabled pylama in checks.

File: 2023Q3.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+== Membership
+
+Since its official establishment on April 12, 2022, SIGTURK's membership has been steadily increasing.
+
+== Board
+
+* President: Duygu Ataman, New York University, USA
+* Secretary: Sardana Ivanova, University of Helsinki, Finland
+
```
Comment (j0ma) [inline 2023Q3.adoc]: Should this include other officers as board members?

File: 2023Q3.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+== Elections
+
+Following the successful completion of the first formal elections in Autumn 2022, SIGTURK also had an election to increase number of officers.
```
Comment (j0ma) [inline 2023Q3.adoc:24]: Can we mention the total number of officers?

File: 2023Q3.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+== Activities
+
+=== Reading and Research Sub Interest Group
+
+In our continued pursuit of fostering a comprehensive academic environment, a Reading and Research Sub Interest Group was established. The group aims to encourage and facilitate the exploration of the rich linguistic landscape of Turkic languages.
```
Comment (j0ma) [inline 2023Q3.adoc]: Maybe we can mention contextualizing Turkic languages within modern NLP

File: 2023Q3.adoc [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+=== Core Specification
+
+SIGTURK continues to iterate on its Core specification to foster an open, participatory, and time-zone inclusive environment. The Core is available at link:++https://sigturk.github.io/sigturk++[https://sigturk.github.io/sigturk].
+
+== Meetings
+
+With a growing international member base, plans are being made to accommodate various timezones by scheduling more meetings.
```
Comment (j0ma) [inline 2023Q3.adoc]: Mention particular conflicts (UTC/daylight savings time/off-by-one errors etc)

File: 2023Q3.adoc [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+== Communications
+
+=== Website
+
+SIGTURK website is at link:++https://sigturk.github.io++[https://sigturk.github.io].
+
+=== Mailing List
+
+SIGTURK members get up-to-date information from the mailing list at link:++https://groups.google.com/g/sigturk++[https://groups.google.com/g/sigturk].
+
```
Comment (j0ma) [inline 2023Q3.adoc:53]: Semi-related to comms: Could we add a publicly viewable Google Calendar? Rn I have to search for the invites in my inbox

File: 2023Q3.adoc [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+== Future
+
+== Annotations and Python Convenience Packages
+
+Looking ahead, SIGTURK plans to explore the annotation of phrasal structures across Turkic languages and develop Python convenience packages, adhering to clean code guidelines, to enhance computational linguistic research accessibility.
+
+=== Workshop on Natural Language Processing for Turkic Languages
+
+SIGTURK intends to host a Workshop on Natural Language Processing for Turkic Languages.
```
Comment (j0ma) [inline 2023Q3.adoc]: Any idea when?

**Review by j0ma — CHANGES_REQUESTED** [review]

File: 2023Q3.adoc (hunk 6 above)
Comment (d-ataman) [reply 2023Q3.adoc]: It's a report so we can state as "SIGTURK started planning the organization of an annual workshop in conjunction with an ACL conference."
As a side note, we will apply and make a preferred list of venues, but final allocation is decided by the executive committee.

File: 2023Q3.adoc (hunk 2 above)
Comment (d-ataman) [reply 2023Q3.adoc:24]: agree

Comment (d-ataman) [reply 2023Q3.adoc:24]: also election outcome " number is increased from 4 to 5. The new position was allocated to Aziza Mirsaidova".

File: 2023Q3.adoc (hunk 1 above)
Comment (mehmetoguzderin) [reply 2023Q3.adoc]: I have included

File: 2023Q3.adoc (hunk 2 above)
Comment (mehmetoguzderin) [reply 2023Q3.adoc:24]: Included

File: 2023Q3.adoc (hunk 5 above)
Comment (mehmetoguzderin) [reply 2023Q3.adoc:53]: I'll take a look at it

File: 2023Q3.adoc (hunk 4 above)
Comment (mehmetoguzderin) [reply 2023Q3.adoc]: Added a little clause

File: 2023Q3.adoc (hunk 3 above)
Comment (mehmetoguzderin) [reply 2023Q3.adoc]: Added

**Review by j0ma — APPROVED** [review]

**Pull request merged**